gradle.properties

```diff
@@ -3,3 +3,5 @@ org.gradle.parallel=true
 jackson.version=2.9.2
 
 includeCloudProviders=all
+
+includeDatadogTracing=false
```

Comment:
false?

Reply:
defaulting to false yes, in case we get this upstream. so the default would be it's off, but if you want to turn it on, run the gradle build with the properties set to true.